amqp/src/main/scala/akka/stream/alpakka/amqp/javadsl/AmqpSource.scala

```diff
@@ -13,6 +13,31 @@ object AmqpSource {
    * Java API: Creates an [[AmqpSource]] with given settings and buffer size.
    */
   def create(settings: AmqpSourceSettings, bufferSize: Int): Source[IncomingMessage, NotUsed] =
```

Comment:
If we are just calling the new one, then maybe `@deprecate` it and tell people to use `atMostOnceSource` instead.

Reply:
100% agree on this one. I was going to ask about that but didn't want to modify much the API without taking first with core maintainers. I'll `@deprecate` the old constructor in favour of `atMostOnceSource `